slidez-fe-core/src/services/ws/ws-helper.ts

```diff
@@ -0,0 +1,50 @@
+import SockJS from 'sockjs-client'
+import Stomp, { Client, Message } from 'webstomp-client'
+
+let socket: WebSocket | undefined = undefined
+let stompClient: Client | undefined = undefined
```

Comment:
Please don't create global variables...
You can use [singleton ](https://stackoverflow.com/questions/1479319/simplest-cleanest-way-to-implement-a-singleton-in-javascript)here